packages/twenty-front/src/modules/object-record/hooks/useRecordTableContextMenuEntries.tsx

```diff
@@ -143,6 +175,15 @@ export const useRecordTableContextMenuEntries = (
         //   Icon: IconNotes,
         //   onClick: () => {},
         // },
+        ...(dataExecuteQuickActionOnmentEnabled
+          ? [
+              {
+                label: 'Quick Action',
```

Comment:
In a following PR we will improve this, with a dropdown of actions driven by backend and being object-dependent